pytype_reporter.py

```diff
@@ -499,23 +509,79 @@ def load_config(config_file: str, script_basename: str) -> Config:
     return config
 
 
-def get_changed_xfail_files(config: Config) -> list[str]:
-    """Get the list of changed files compared to the default branch.
-
-    :param config: The configuration dictionary.
-    :return: The list of changed files.
-    """
+def git_diff(*args) -> str:
+    """Run git diff with the given arguments and return the output as a string."""
     return check_output(  # nosec:B603
         args=[
             "git",
             "diff",
             "--ignore-space-change",
-            "--name-only",
-            "origin/" + config["default_branch"],
-            *config["expected_to_fail"],
+            *args,
         ],
         universal_newlines=True,
-    ).splitlines()
+    )
+
+
+def find_branch_point(config):
+    """Get the commit hash of the default where the current branch was created from"""
+    cmd = ["git", "merge-base", "origin/" + config["default_branch"], "HEAD"]
+    return check_output(cmd, universal_newlines=True).strip()  # nosec:B603
+
+
+def github_get_pr_commit_messages(repo, pr_number):
+    """Get the commit messages of the PR from the GitHub API"""
+    empty = []
+    github_token = env.get("GITHUB_TOKEN")
+    debug("GITHUB_TOKEN: %s", github_token)
```

Comment:
Do not print the token it's a secret!

Reply:
GitHub of course filters the Token in the log of course, I checked.

But I can/will (asap) remove it because it is not needed to print it.